roles/kubernetes-apps/istio/tasks/istio

```diff
@@ -0,0 +1,33 @@
+#! /bin/sh
```

Comment:
I'm not seeing this file get used/called?

Reply:
this should be removed. thanks for pointing it out